### PR TITLE
Tactical anim update

### PR DESCRIPTION
**Pull request by Scrndles**
This ScrnBalance version currently requires the new ScrnAnims file to compile/run, ScrnMP5MBullets.uc has Mesh=SkeletalMesh'ScrnAnims.mp5m_bullets' since my attempt to load it dynamically via MeshRef doesn't work.

Here are the changes in this version:
Added tactical reload to Dualies, Dual HCs and Dual MK23s
Added new animation for MP5M tactical reload
Added new animation for Boomstick single shell reload

### CODE REVIEW

File: Classes/ScrnDeagle.uc [hunk 1]
```diff
@@ -33,7 +39,7 @@ var rotator PistolHammerRotation; //for deagle's stupid hammer
 
 simulated function BringUp(optional Weapon PrevWeapon)
 {
-    Super.BringUp(PrevWeapon);
+	Super.BringUp(PrevWeapon);
```
Comment (poosh) [inline Classes/ScrnDeagle.uc]: Once again, tabs sneaked into the code :)

File: Classes/ScrnDeagleFire.uc [hunk 2]
```diff
@@ -7,26 +7,23 @@ var bool  bCheck4Ach;
...
+        if( (Level.TimeSeconds - LastFireTime > FireRate) && !KFWeapon(Weapon).bIsReloading )
         {
-            ScrnDeagle(Weapon).LockSlideBack(); //lock slide back
-            ScrnDeagle(Weapon).RotateHammerBack(); //rotate hammer back
+            if(KFWeapon(Weapon).MagAmmoRemaining <= 1  )
+            {
+                ScrnDeagle(Weapon).LockSlideBack(); //lock slide back
+                ScrnDeagle(Weapon).RotateHammerBack(); //rotate hammer back
+            }
+            if(KFWeapon(Weapon).MagAmmoRemaining > 1  )
```
Comment (poosh) [inline Classes/ScrnDeagleFire.uc]: This "if" can be replaced with "else"

File: Classes/ScrnDualDeagleFire.uc [hunk 3]
```diff
@@ -11,6 +11,62 @@ function DoFireEffect()
...
+{
+    //this gets toggled before firing, amazingly
+    if (ScrnDualDeagle(Weapon).MagAmmoRemaining%2 == 1)
+    {
+        FireAnim2 = default.FireAnim;
+        FireAimedAnim2 = default.FireAimedAnim;
+        FireAnim = default.FireAnim2;
+        FireAimedAnim = default.FireAimedAnim2;
+    }
+    if (ScrnDualDeagle(Weapon).MagAmmoRemaining%2 == 0)
```
Comment (poosh) [inline Classes/ScrnDualDeagleFire.uc]: Replace it with `else`

File: Classes/ScrnDualMK23Fire.uc [hunk 4]
```diff
@@ -3,6 +3,50 @@ class ScrnDualMK23Fire extends DualMK23Fire;
...
+{
+    //this gets toggled before firing, amazingly
+    if (ScrnDualMK23Pistol(Weapon).MagAmmoRemaining%2 == 0)
+    {
+        FireAnim2 = default.FireAnim;
+        FireAimedAnim2 = default.FireAimedAnim;
+        FireAnim = default.FireAnim2;
+        FireAimedAnim = default.FireAimedAnim2;
+    }
+    if (ScrnDualMK23Pistol(Weapon).MagAmmoRemaining%2 == 1)
```
Comment (poosh) [inline Classes/ScrnDualMK23Fire.uc]: else

File: Classes/ScrnDualiesFire.uc [hunk 5]
```diff
@@ -1,5 +1,58 @@
...
+{
+    //this gets toggled before firing, amazingly
+    if (ScrnDualies(Weapon).MagAmmoRemaining%2 == 1)
+    {
+        FireAnim2 = default.FireAnim;
+        FireAimedAnim2 = default.FireAimedAnim;
+        FireAnim = default.FireAnim2;
+        FireAimedAnim = default.FireAimedAnim2;
+    }
+    if (ScrnDualies(Weapon).MagAmmoRemaining%2 == 0)
```
Comment (poosh) [inline Classes/ScrnDualiesFire.uc]: else

File: Classes/ScrnMK23Pistol.uc [hunk 6]
```diff
@@ -7,45 +7,49 @@ var transient bool  bShortReload;
...
-    if (MagAmmoRemaining == 0)
-        LockSlideBack();
-}
-
-simulated function ResetSlidePosition()
-{
-    SetBoneLocation( 'Slide', PistolSlideOffset, 0 ); //reset slide position
+    if (Level.NetMode != NM_DedicatedServer)
+    {
+        Super.BringUp(PrevWeapon);
```
Comment (poosh) [inline Classes/ScrnMK23Pistol.uc]: Shouldn't super call be moved outside the dedicated server check (i.e., called on dedicated server too)?

**Review by poosh — APPROVED** [review]
Amazing work! Fix mentioned minor code issues and I'll merge it.

File: Classes/ScrnMK23Pistol.uc (hunk 6 above)
Comment (Scrndles) [reply Classes/ScrnMK23Pistol.uc]: oops, this was a bad copypaste

**Review by poosh — APPROVED** [review]

**Pull request merged**